ERCS/erc-7250.md

```diff
@@ -0,0 +1,160 @@
+---
+eip: 7250
```

Comment:
```suggestion
eip: 7824
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.

Reply:
I applied the change of number as suggested,
I decided to split into two ERC
- 7824: Define the EXIT format: https://github.com/statechannels/exit-format
- 7825: Define the State Channels types: https://docs.statechannels.org/protocol-tutorial/0010-states-channels/

Please note this code was audited by Consensys Diligence